### PR TITLE
Quick Start Enhancement

### LINKED ISSUE #972: Quick Start Enhancement
Quick Start should be enhanced such that:
- It is more obvious that it is optional to take the quick start route.
- Add new information to summary page if user arrives there through quick start.

### PR DESCRIPTION
**Pull request by streetcoder123**
Task: https://microsoftgarage.visualstudio.com/Intern%20GitHub/_workitems/edit/31914

**Changes**
- Changed the text above the quick start button to make it more apparent that this is an optional route.
- Made the Quick Start button a little smaller
- Added state for the text on summary page so that it is different from when a user goes through quick start than when the user goes through the complete wizard flow.

closes #972

### CODE REVIEW

**Comment by SahilTara** [comment]
@njindal239 
image

Seems to be some visual glitch here

File: src/client/src/containers/ReviewAndGenerate/index.tsx [hunk 1]
```diff
@@ -31,19 +35,27 @@ const messages = defineMessages({
     id: "context.almostDone",
     defaultMessage:
       "You're almost done - review your project details on the right and make any necessary adjustments!"
+  },
+  quickStart: {
+    id: "context.quickStart",
+    defaultMessage: `Woo-hoo! You skipped to the end where we'll get you started quickly. Just click ""Create Project".`
```
Comment (SahilTara) [inline src/client/src/containers/ReviewAndGenerate/index.tsx]: image

**Review by SahilTara — COMMENTED** [review]
Actually the whole page seems to be messed up for me, (this is the highest i can scroll). Not sure if this is a issue related to your PR or if this should be a new issue yet, will need to check dev. 
image

**Comment by streetcoder123** [comment]
> @njindal239
> image
> 
> Seems to be some visual glitch here

This issue has been logged in the issues already. https://github.com/microsoft/WebTemplateStudio/issues/974

I wonder why is this footer yellow though

**Comment by jasmingoh** [comment]
> @njindal239
> 
> Seems to be some visual glitch here

Hey @SahilTara, what VS Code version are you using?

**Comment by SahilTara** [comment]
@njindal239 Don't worry about yellow footer that's just my theme haha.

**Comment by jasmingoh** [comment]
> Actually the whole page seems to be messed up for me, (this is the highest i can scroll). Not sure if this is a issue related to your PR or if this should be a new issue yet, will need to check dev.

@SahilTara, @njindal239 was talking to me about the scrolling. It seems like we cannot reproduce it.. do you mind letting us know how to ?

**Comment by SahilTara** [comment]
> > Actually the whole page seems to be messed up for me, (this is the highest i can scroll). Not sure if this is a issue related to your PR or if this should be a new issue yet, will need to check dev.
> 
> @SahilTara, @njindal239 was talking to me about the scrolling. It seems like we cannot reproduce it.. do you mind letting us know how to ?

Honestly all i do is start webts. May have to do with screensize (I'm on my windows computer right now). 


image

Think it has to do with the scaling of 125%. 

image

@jasmingoh

**Comment by jasmingoh** [comment]
@SahilTara Ok, I'm able to reproduce this now. Seems like it is also happening on dev.. could you create an issue for it? Thanks!

**Review by annielovesjs — CHANGES_REQUESTED** [review]
use` z-index: 1000 `on the footer to not have the quickstart button overlap

**Comment by streetcoder123** [comment]
> use`z-index: 1000`on the footer to not have the quickstart button overlap

@annielovesjs This issue has already been logged under issues (#974 ) and is not caused by this PR. I shall address it as and when that issue gets addressed, probably next Sprint.

File: src/client/src/containers/ReviewAndGenerate/index.tsx (hunk 1 above)
Comment (SahilTara) [reply src/client/src/containers/ReviewAndGenerate/index.tsx]: @njindal239  You seem to have an extra " here

Comment (streetcoder123) [reply src/client/src/containers/ReviewAndGenerate/index.tsx]: Whoops! Thanks

**Review by annielovesjs — APPROVED** [review]

**Comment by jasmingoh** [comment]
Just for record, the issue @SahilTara brought up is logged here #991

**Review by jasmingoh — APPROVED** [review]

**Review by piotrmark — APPROVED** [review]

**Pull request merged**